src/benchmarks/micro/corefx/System.Text.Json/Utf8JsonWriter/Perf.Base64.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using System.Linq;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Extensions;
+using MicroBenchmarks;
+
+namespace System.Text.Json.Tests
+{
+    [BenchmarkCategory(Categories.CoreFX, Categories.JSON)]
+    public class Perf_Base64
+    {
+        private ArrayBufferWriter<byte> _arrayBufferWriter;
+        private byte[] _data;
+        private byte[] _dataWithNoEscaping;
+        private byte[] _dataWithEscaping;
+
+        [Params(10, 100, 1000)]
+        public int NumberOfBytes { get; set; }
+
+        [Params(true, false)]
+        public bool Formatted;
+
+        [Params(true, false)]
+        public bool SkipValidation;
+
+        [GlobalSetup]
+        public void Setup()
+        {
+            _arrayBufferWriter = new ArrayBufferWriter<byte>();
+            _data = ValuesGenerator.Array<byte>(NumberOfBytes);
+
+            // Results in a number of A plus padding
+            _dataWithNoEscaping = new byte[NumberOfBytes];
+
+            // Results in a lot + and /
+            _dataWithEscaping = Enumerable.Repeat(0, NumberOfBytes)
```

Comment:
Instead of `Repeat` this should be `Range`. Otherwise it is always `0xFB`. I'll update this.

Reply:
Will it still result in lots of `+` and `/`?

Reply:
Yes. The `%` makes it `0xFB` or `0xFF`. I had to intention to use `Range`, but somehow I typed `Range`, so it resulted in repeating `0`, hence the `%` was effectless.